docs/quality-assurance-and-test.md

```diff
@@ -4,43 +4,147 @@ Welcome to the introduction on how we are working with Quality Assurance and Tes
+
+Building the right product has in earlier times been proven by a post-developing test - We want to have this mitigation before implementing, as a way of ensuring alignment of expectations. Thus the knowlegde and the learnings of this risk evaluation could also be used for automating regression test within the pipeline so we remember to re-visit our acceptance criteria throughout the delivery pipeline.
+
+Regarding the second question on how we are delevoping our solutions. Here we talk about issues that could affect the way we are able to deliver a solution. Considering complexity, knowledge levels, tools, experience etc. is important, as these risks would very easily impact the work needed and the quality of the solution.
+
+*Are you working on a product/solution and are you unsure of the risks - or how to mitigate certain risks you have identified? - use the community for collaboration and discussion.*
+
+### Working with risk
+
+One way of mitigating risks is testing - but we are all familiar with the old saying that test is a timeboxed event/a bottleneck before go live - In Green Energy we wish to ensure that test/QA is happening all the time, by everyone. We don't have a verification team - we are all responsible for mitigating risks, and ensuring quality - so rest assure that it is never too early to talk quality.
 
 ### Testing Quadrants
 
-A way of mitigating risk factors.
+One way of translating risk items to actual testing could be using the testing quadrants for identifying - the relevant test types for specific areas.
+
+[image]
 
 ### Risk Radar
 
 A way of visualizing the migitation of risks.
 
-## Test Strategy Elements
+[TBW]
+
+## Open Source and Test Approach Elements
+
+This section is meant as inspiration for all contributors to Green Energy Hub. Whether you're working on forked instance or delivering into the community - we would like to share the thoughts, and choices we have made to ensure QA and testing activities througout our work with Green Energy Hub. Some of the choices and activities are done to ensure an alignment towards our community and Green Energy Hub as a whole, and some activities are done in the Energinet setup.
+
+### Green Energy Hub - Working with issue templates
+
+We are securing the quality of issue-submission by using templates for different kinds of issues – please choose the template you need, when creating an issue.
+
+The life cycle of an issue is dynamic, and the commit team and community in general will handle the life span of an issue, according to activity for these. Please remember that all issues can be re-opened if needed.  
+
+Please use domain repos and process tagging (e.g. change of energy supplier) on issues for enhancing traceability, transparency, and history.
+
+### Green Energy Hub - Content of Pull Requests
+
+A Pull Request should always be,
+
+- Related to an accepted issue
```

Comment:
That might be - but you had no task related to the work being done - I'm guessing if you ask you SM or PO, it would be appreciated to have a linked issues for all in your board no matter the size of the task - rather than not having the task at all.